src/app/features/dashboard/dashboard.component.ts

```diff
@@ -115,18 +115,24 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
 
     this.menuItems = this.authService.currentUser.pipe(
       map((token) => token!.roles),
-      map((roles) => roles.map((role) => ({ routerLink: [`/arpa/dashboard/${role}`], label: role.toUpperCase() })))
```

Comment:
Ich finde es ehrlich gesagt nicht gut dieses Feature in die dashboard component zu packen. Die dashboard component ist als Grundgerüst für alle Roles gedacht, die spezifischen Implementierungen sind separat. Außerdem gilt dieses Feature nur für Performer. Ich würde deshalb ein neues Widget anlegen, dort die Logik reinbauen und das dann dem performer dashboard zuweisen.

Reply:
Absolut dafür, danke für den Hinweis. Traue mir das nur im Moment noch nicht selbst zu da noch andere Probleme auftauchen wie, daß bspw. "ComponentFactoryResolver" als "deprecated" markiert ist und noch so was in der Art weiter unten. Ich würde das lieber zusammen mit @dmpasilva angehen am Wochenende (falls er Zeit hat) und dann erst das ganze in separates Widget packen, wenn das drumrum auch geklärt ist. "ngComponentOutlet" könnte evtl. die Lösung sein, blick aber noch nicht ganz durch. 😬